save-cloud-common/src/jvmMain/kotlin/org/cqfn/save/entities/LnkUserProject.kt

```diff
@@ -14,11 +14,11 @@ import javax.persistence.ManyToOne
 class LnkUserProject(
     @ManyToOne
     @JoinColumn(name = "project_id")
-    var project: Project?,
```

Comment:
Will we have some problems with hibernate or mapping?

Reply:
Yes, this 100% should be nullable to allow lazy loading

Reply:
Ok. Fixed